targets/TARGET_TT/TARGET_TT_M4G9/Periph_Driver/inc/adc_ch.h

```diff
@@ -0,0 +1,135 @@
+/** 
+ *******************************************************************************
+ * @file    adc_ch.h
+ * @brief   This file provides all the functions prototypes for ADC driver. \n
+ *          Channel Class.
+ * @version V1.0.0.0
+ * $Date:: 2017-09-07 13:52:12 #$
+ * 
+ * DO NOT USE THIS SOFTWARE WITHOUT THE SOFTWARE LICENSE AGREEMENT.
+ * 
+ * (C)Copyright TOSHIBA MICROELECTRONICS CORPORATION 2017 All rights reserved
```

Comment:
I dont see the license, neither agreement should be required? Can you please update peripheral driver license to be compatible with our licensing guidance ? Recommend using apache 2.0 as for the rest of files in this pull request